golangci.yaml

```diff
@@ -181,7 +181,7 @@ linters-settings:
         #     reason: "testing if blocked version constraint works."  # Reason why the version constraint exists. (Optional)
   govet:
     # report about shadowed variables
-    check-shadowing: true
+    # check-shadowing: true
```

Comment:
Could you please explain why this is commented?  Is it b/c it's always enabled by default or you want to turn it off?
In the doc it says:
```
Shadowed variables [¶](https://pkg.go.dev/github.com/golangci/govet#hdr-Shadowed_variables)
Flag: -shadow=false (experimental; must be set explicitly)

Variables that may have been unintentionally shadowed.
```

Reply:
This is my suggestion. 
`check-shadowing`'s default value is false, and when it's set to true, more warnings are reported. Most of them are false positives.